datasource/hcp-packer-image/data.go

```diff
@@ -201,6 +197,7 @@ func (d *Datasource) Execute() (cty.Value, error) {
 					CreatedAt:     image.CreatedAt.String(),
 					BuildID:       build.ID,
 					IterationID:   build.IterationID,
+					ChannelID:     channelID,
```

Comment:
Indentation seems off here 🤔 

That said the output value of ChannelID is only known if a channel was provided as an input to the datasource correct? 

If so, this needs to be documented to ensure a user does not rely on the channelID output always being available.

Reply:
I ran `go fmt`on these, maybe is the diff that is messing it 🤔 
Good point on documenting it, I will do it!